go

```diff
@@ -64,8 +64,12 @@ def update_gems
   exec_cmd 'bundle'
 end
 
+def update_submods
+  edec_cmd "git submodule update --remote"
+end
+
 def update_data
-  exec_cmd 'cd _data && ./import-public.rb'
+  exec_cmd '_data/import-public.rb'
```

Comment:
at 71b2f36 I did it this way, do you think it will work?
```
def update_data
  ruby = exec_cmd 'which ruby'
  exec_cmd "#{ruby} import-public.rb"
end
```
Trying to write it in such a way that if `ruby` ever moves the script will still work.